apps/communications/dialer/test/marionette/call_log_test.js

```diff
@@ -1,78 +0,0 @@
-'use strict';
```

Comment:
Like for `apps/costcontrol/test/marionette/lib/costcontrol.js`, no need to delete this file :wink: